src/styles/index.scss

```diff
@@ -1,3 +1,84 @@
-body {
+@import './utils/variables';
+
+* {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
```

Comment:
It's a good practice to set the font-size in the body to 16px (which is the default browser font-size) and use rem units for other font-sizes in your CSS. This way, you're making your website accessible to users who change their default browser font-size. You can set the font-size of specific elements relative to this base font-size. For example, if you want to set the font-size of an element to 30px, you can set it to 1.875rem (because 30 / 16 = 1.875).